tests/client/rpc_client.h

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <rpc.h>
+
+RPC_STATUS
+initialize_rpc_binding();
```

Comment:
```suggestion
initialize_rpc_binding(void);
```
Same in line 12.  See https://softwareengineering.stackexchange.com/questions/286490/what-is-the-difference-between-function-and-functionvoid#:~:text=In%20C%2C%20a%20function%20with%20the%20parameter%20list,means%20%22no%20parameters%22%20the%20same%20as%20void%20does. for details